src/classes/ctx.js

```diff
@@ -74,6 +74,22 @@ module.exports = class Ctx {
     this._client.sendMessage(jid, content, options);
   }
 
+  async sendAudio(url, vn) {
+      this._client.sendMessage(this.id, { audio: { url: url }, mimetype: 'audio/mp4', ptt: vn || false }, { quoted: this._msg })
```

Comment:
i think you can add new jid parameter and pass it into sendMessage (like ctx.sendMessage function)
 
 e.g:
```js
ctx.sendAudio(jid, url, vn)
// etc
```

and i think you should remove the quoted, or add new parameter to send it with quote or not

Reply:
and with other functions too

Reply:
ok